src/ResourceManager/Network/Commands.Network/ApplicationGateway/ListAzureApplicationGatewaySslPredefinedPolicy.cs

```diff
@@ -0,0 +1,57 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Network
+{
+    using AutoMapper;
+    using Microsoft.Azure.Commands.Network.Models;
+    using Microsoft.Azure.Management.Network;
+    using System.Collections.Generic;
+    using System.Management.Automation;
+
+    [Cmdlet(VerbsCommon.Get, "AzureRmApplicationGatewaySslPredefinedPolicy"), 
+        OutputType(typeof(PSApplicationGatewaySslPredefinedPolicy), typeof(IEnumerable<PSApplicationGatewaySslPredefinedPolicy>))]
+    [Alias("List-AzureRmApplicationGatewaySslPredefinedPolicy")]
+    public class GetAzureApplicationGatewaySslPredefinedPolicy : ApplicationGatewayBaseCmdlet
+    {
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Name of the ssl predefined policy")]
+        [ValidateNotNullOrEmpty]
+        public string PredefinedPolicyName { get; set; }
```

Comment:
@akshaysngupta can you change the name of this parameter to `Name`? You can still add an alias to `PredefinedPolicyName`